delphi/translators/GrFN2WiringDiagram/SIR-Gillespie-SD__alt_lambdas.py

```diff
@@ -0,0 +1,259 @@
+import math
```

Comment:
Similarly for this file - can it be automatically generated? If so, let's remove it.